modin/core/execution/ray/implementations/pandas_on_ray/partitioning/virtual_partition.py

```diff
@@ -293,26 +312,39 @@ def apply(
             In this case, we have to return the partitioning to its previous
             orientation (the lengths will remain the same). This is ignored between
             two axis partitions.
+        lengths : list, optional
+            The list of lengths to shuffle the object.
+        manual_partition : bool, default: False
+            If True, partition the result with `lengths`.
         **kwargs : dict
             Additional keywords arguments to be passed in `func`.
 
         Returns
         -------
         list
             A list of `PandasOnRayDataframeVirtualPartition` objects.
+
+        Notes
+        -----
+        The ``args`` iterable is deserialized before being passed to the superclass ``apply``
+        method. In older versions of Modin, ``args`` was instead passed internally as
+        ``kwargs["args"]``, and this deserialization was handled in a special case in
+        ``deploy_ray_func``, making control flow difficult to follow.
```

Comment:
Is this actual?

Reply:
`args` isn't deserialized here (this was left over from when I was messing around with deserializing in different spots), and instead deserialization takes place in `deploy_ray_func` as before. However, it is true that we were previously passing positional arguments as `kwargs["args"]` (you can see it in the diff, and [here](https://github.com/modin-project/modin/blob/7c260ad55d40bd31de27a21c66f47202e430b66e/modin/core/execution/ray/implementations/pandas_on_ray/partitioning/virtual_partition.py#L310)).

Reply:
I've changed the comments to reflect this.